pkg/apis/agones/v1/fleet_test.go

```diff
@@ -155,6 +155,19 @@ func TestFleetGameserverSpec(t *testing.T) {
 	causes, ok = f.Validate()
 	assert.False(t, ok)
 	assert.Len(t, causes, 2)
+
+	f = defaultFleet()
+	f.ApplyDefaults()
+	f.Spec.Template.ObjectMeta.Labels = make(map[string]string)
+	nameLen := validation.LabelValueMaxLength + 1
+	key := make([]byte, nameLen)
+	for i := 0; i < nameLen; i++ {
+		key[i] = 'f'
+	}
+	f.Spec.Template.ObjectMeta.Labels["label"] = string(key)
```

Comment:
Should be able to just do `f.Spec.Template.ObjectMeta.Labels["label"] = strings.Repeat("f", validation.LabelValueMaxLength + 1)`

See https://golang.org/pkg/strings/#Repeat